build.gradle.kts

```
@@ -0,0 +1,23 @@
+import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
+import org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask
+
+plugins {
+    alias(libs.plugins.kotlinx.binaryCompatibilityValidator)
+}
+
+val deploy by tasks.creating() {
```

```suggestion
val deploy by tasks.creating {
```

unnecessary brackets

---

fixed